From c782dcee7405044bf4b3513f7ad5b2ca22977c67 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Tue, 31 Jul 2012 03:25:18 +0300
Subject: [PATCH 270/311] confc.c (wait_reply_st_in): remove misleading comment

We don't need blocking semantics within wait_reply_st_in().

RB: r/939/diff/1/?file=25263#file25263line679
---
 conf/confc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/conf/confc.c b/conf/confc.c
index 52a2f7a..0756b91 100644
--- a/conf/confc.c
+++ b/conf/confc.c
@@ -665,7 +665,6 @@ static int wait_reply_st_in(struct c2_sm *mach)
 
 	C2_PRE(request_is_valid(ctx->fc_req));
 
-	/* XXX Consider using c2_rpc_client_call() instead of c2_rpc_post(). */
 	rc = c2_rpc_post(&ctx->fc_fop.f_item);
 	if (rc == 0)
 		return -1;
-- 
1.8.3.2

